### PR TITLE
feat: Lossless JSON import

### LINKED ISSUE #6174: `JSONImportTask` should not go through Markdown converter
Follow on from https://github.com/outline/outline/pull/5553 – we need to avoid the markdown translation to avoid lossyness going forward.

### PR DESCRIPTION
**Pull request by hmacr**
Closes #6174

### CODE REVIEW

File: server/queues/tasks/ImportJSONTask.ts [hunk 1]
```diff
@@ -152,21 +153,79 @@ export default class ImportJSONTask extends ImportTask {
...
+      });
+
+      return Fragment.fromArray(nodes);
+    };
+
+    for (const document of output.documents) {
+      const node = Node.fromJSON(schema, document.data);
+      const transformedNode = node.copy(transformFragment(node.content));
+      document.data = transformedNode;
+      document.text = serializer.serialize(transformedNode);
```
Comment (tommoor) [inline server/queues/tasks/ImportJSONTask.ts:228]: Do we end up serializing the node twice here, once on L81 and then again?

Comment (hmacr) [reply server/queues/tasks/ImportJSONTask.ts:228]: Umm.. yes.
I think this `for` loop should be moved inside a `if (output.attachments.length) { /* logic to replace attachment urls */ }`.

Comment (hmacr) [reply server/queues/tasks/ImportJSONTask.ts:228]: I was thinking that I could make this change once the initial review is over for this PR.
Please let me know when you've reviewed it.

Comment (tommoor) [reply server/queues/tasks/ImportJSONTask.ts:228]: Yep, reviewed – this is the only thing that stood out.

**Comment by hmacr** [comment]
@tommoor Merge conflicts have been resolved, please check.

**Pull request merged**